src/test/java/com/squareup/javapoet/MethodSpecTest.java

```diff
@@ -151,9 +176,7 @@ private ExecutableElement findFirst(Collection<ExecutableElement> elements, Stri
     TypeElement classElement = getElement(ExtendsIterableWithDefaultMethods.class);
     DeclaredType classType = (DeclaredType) classElement.asType();
     List<ExecutableElement> methods = methodsIn(elements.getAllMembers(classElement));
-    ExecutableElement exec = findFirst(methods, "iterator");
-    assume().that(Util.DEFAULT).isNotNull();
```

Comment:
Why is `Util.DEFAULT` still a thing? Shouldn't it be inlined everywhere now as `Modifier.DEFAULT`?

Reply:
#617

Reply:
Ah I thought that had landed